meta-iot2000-bsp/README.md

```diff
@@ -11,18 +11,18 @@ This layer depends on:
 ```
   URI: git://git.yoctoproject.org/poky
   layers: meta, meta-yocto, meta-yocto-bsp
-  branch: krogoth
+  branch: morty
 
   URI: git://git.yoctoproject.org/meta-intel
   layers: meta-intel
-  branch: krogoth
+  branch: morty
 ```
 
 
 Building the meta-iot2000 BSP Layer
 ===================================
 
-This uses Yocto 2.1 (Krogoth) with the 4.4 Linux kernel provided by Yocto and
+This uses Yocto 2.2 (Morty) with the 4.8 Linux kernel provided by Yocto and
```

Comment:
Hm, you're selecting the 4.4 kernel as preferred in commit "bsp: Select the 4.4 yocto kernel" while saying that 4.8 of Morty is used here?